src/System.Web.Http/Validation/Validators/DataAnnotationsModelValidator.cs

```diff
@@ -21,6 +29,13 @@ public DataAnnotationsModelValidator(IEnumerable<ModelValidatorProvider> validat
             Attribute = attribute;
         }
 
+        // Internal for testing
+        internal static bool UseLegacyValidationMemberName
```

Comment:
`static` only because an instance field for testing seemed wasteful.

I doubt it's necessary but could make this property `public` for Microsoft.AspNet.WebApi.Owin users, whose applications are less likely to involve `System.Configuration`. Could even move this (still `static`) into `HttpConfiguration` for high visibility. Thoughts?

Reply:
Naah it's fine the way it is.